src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -794,13 +793,12 @@ private IConversionExpression CreateBoundAsOperatorOperation(BoundAsOperator bou
             Lazy<IOperation> operand = new Lazy<IOperation>(() => Create(boundAsOperator.Operand));
             SyntaxNode syntax = boundAsOperator.Syntax;
             Conversion conversion = boundAsOperator.Conversion;
-            bool isExplicit = true;
             bool isTryCast = true;
             bool isChecked = false;
             ITypeSymbol type = boundAsOperator.Type;
             Optional<object> constantValue = ConvertToOptional(boundAsOperator.ConstantValue);
-            bool isImplicit = boundAsOperator.WasCompilerGenerated;
-            return new LazyCSharpConversionExpression(operand, conversion, isExplicit, isTryCast, isChecked, _semanticModel, syntax, type, constantValue, isImplicit);
+            bool isImplicit = false; // as expressions can never be implicitly created
```

Comment:
>// as expressions can never be implicitly created [](start = 37, length = 49)

That wouldn't be true for lowered tree. I would keep the original code.

Reply:
Done.